### PR TITLE
Fixed a bug where images of the welcome board were not copied over.

### LINKED ISSUE #2451: Bug: Duplicating Welcome to Boards board from sidebar doesn't work
## Steps to reproduce the behavior

1. Once done with the product tour, try duplicating the board from the left sidebar.
2. Notice the board doesn't duplicate.

## Expected behavior

User should be able to duplicate the welcome board.

## Edition and Platform

 - Edition: Personal Mattermost plugin
 - Version: 0.15-RC1
 - Browser and OS: Firefox

## Additional context

It only occurs on the welcome board created automatically for the tour.

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Fixed a bug where images of the welcome board were not copied over.

#### Ticket Link
Fixes #2451

### CODE REVIEW

File: server/app/onboarding.go [hunk 1]
```diff
@@ -85,6 +85,10 @@ func (a *App) createWelcomeBoard(userID, workspaceID string) (string, error) {
 
 	blocks = model.GenerateBlockIDs(blocks, a.logger)
 
+	if updateFileIDsErr := a.CopyCardFiles(onboardingBoardID, workspaceID, blocks); updateFileIDsErr != nil {
```
Comment (wiggin77) [inline server/app/onboarding.go:88]: There will likely be a linter error here... error vars should start with "err".

Comment (harshilsharma63) [reply server/app/onboarding.go]: The linter didn't fail but I've renamed it anyways.

Comment (wiggin77) [reply server/app/onboarding.go]: Thanks.  We have not enabled the linter for CI yet.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by jespino — APPROVED** [review]
LGTM

**Comment by harshilsharma63** [comment]
@ogi-m can you test this once before I cherrypick it?

**Comment by harshilsharma63** [comment]
Verified by @ogi-m and cherrypicked into `release-0.15`.

**Pull request merged**